<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seev::NavWidget</class>
 <widget class="QWidget" name="seev::NavWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="newSearchBox">
     <property name="title">
      <string>New Search</string>
     </property>
     <layout class="QGridLayout">
      <item row="2" column="2">
       <widget class="QCheckBox" name="showAdvChk">
        <property name="text">
         <string>Show Advanced</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="seev::OriePredSelector" name="predWidg" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="openStartPathDlgBut">
        <property name="text">
         <string>Edit Start Paths</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="advOptBox">
        <property name="title">
         <string>Advanced Options</string>
        </property>
        <layout class="QFormLayout" name="_2">
         <item row="0" column="0">
          <widget class="QLabel" name="__z1">
           <property name="text">
            <string>seev Conf Path</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="seevConfPathBut">
           <property name="text">
            <string>Select Config Path</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Scan Filesystem Interval</string>
           </property>
           <property name="buddy">
            <cstring>updIntSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="updIntSpin">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
           <property name="singleStep">
            <number>300</number>
           </property>
           <property name="value">
            <number>3600</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="openConfWidgBut">
           <property name="text">
            <string>Edit `orient` Settings</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="typeOrieCmdBut">
           <property name="text">
            <string>Type `orient` Command (Expert)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="orieCmdEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="initSearchBut">
        <property name="text">
         <string>Start Search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="searchEditDlgBut">
        <property name="text">
         <string>Edit Search in a Separate Dialog...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="updateDbBut">
        <property name="text">
         <string>Scan Filesystem</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="__z2">
     <property name="text">
      <string>Saved Searches - Double Click to Load</string>
     </property>
     <property name="buddy">
      <cstring>savedSearchLst</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="savedSearchLst"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>seev::OriePredSelector</class>
   <extends>QWidget</extends>
   <header>widgets/oriepredselector.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showAdvChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>advOptBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
